src/index.js

```diff
@@ -71,3 +71,12 @@ exports.marshalPrivateKey = (key, type) => {
 
   return key.bytes
 }
+
+exports.randomBytes = (number) => {
```

Comment:
This already exists: 
- node: https://github.com/libp2p/js-libp2p-crypto/blob/master/src/crypto/rsa.js#L58
- browser: https://github.com/libp2p/js-libp2p-crypto/blob/master/src/crypto/rsa-browser.js#L52

you just need to expose it